pkgs/development/python-modules/magika/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, click
+, fetchPypi
+, numpy
+, onnxruntime
+, poetry-core
+, python-dotenv
+, pythonOlder
+, tabulate
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "magika";
+  version = "0.5.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-r6C7iDCG/o3JEvweQGb4upr+LuHvmNtkwtduZGehCsc=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    click
+    numpy
+    onnxruntime
+    python-dotenv
+    tabulate
+    tqdm
+  ];
+
```

Comment:
we should consider adding version test, it would catch issues like magika not runing with python 3.11

https://nixos.org/manual/nixpkgs/stable/#tester-testVersion

this doens't need to happen part of this PR

Reply:
I'll add it to make sure we can support all Python versions.